### ISSUE TITLE
FileName rule false positive

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#901]
We use `.rb` as a template handler and get a rubocop violation on our templates:

```
app/views/fields/show.json.rb:1:1: C: FileName: Use snake_case for source file names.
```
